repository/src/main/resources/db/changelog/db.changelog-rdm-2676.xml

```diff
@@ -0,0 +1,11 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="rdm-2676" author="tobias.derose@hmcts.net">
+        <update tableName="field_type">
+            <column name="regular_expression" value="(?:^[0-9]{16}$|^\d{4}-\d{4}-\d{4}-\d{4}$)" />
+            <where> id = 28 </where>
```

Comment:
are we sure on all the databases the id will always be 28? 
Perhaps we can find a better way to find the row to update ?

Reply:
yes !! thanks this is a work in progress pr, I make it pr without checking

Reply:
DONE.